ReflectometryServer/movement_strategy.py

```diff
@@ -159,6 +159,14 @@ def get_distance_relative_to_beam(self, beam):
         """
         return self._displacement - self._dist_along_axis_from_zero_to_beam_intercept(beam)
 
+    def get_distance_relative_to_beam_in_mantid_coordinates(self, incoming_beam):
+        """
+        Returns (ReflectometryServer.geometry.Position): distance to the beam in mantid coordinates as a vector
+        """
+        distance_relative_to_beam = self.get_distance_relative_to_beam(incoming_beam)
+        return Position(distance_relative_to_beam * sin(radians(self._angle)),
+                        distance_relative_to_beam * cos(radians(self._angle)))
```

Comment:
Feel like this is duplicating code from `position_in_mantid_coordinates` - perhaps we could have a more generic function that takes two points on the movement axis and returns a vector in mantid coordinates, then call that with appropriate arguments where we get a vector for 
- current displacement (from zero)
- beam intercept (from zero)
- offset (from intercept)

etc.

Reply:
I disagree with the above analysis. I think it is important though so I will document on the wiki.

Reply:
After discussion we want the detector offset readback to be based on fixed theta. Everything else is correct.